cmd/modern/root/config/add-user.go

```diff
@@ -59,7 +67,7 @@ func (c *AddUser) DefineCommand(...cmdparser.CommandOptions) {
 	c.AddFlag(cmdparser.FlagOptions{
```

Comment:
>cmdparser.FlagOptions{ [](http://example.com/codeflow?start=11&length=22)

do we have an issue open to enable entering the password through standard input or by being prompted for it? sqlcmd users are generally accustomed to the password prompt in interactive mode, and ci/cd pipelines that use Azure CLI can pipe a password to `az devops login` directly using stdin. We could offer both models as well.

Reply:
Entered:

[https://github.com/microsoft/go-sqlcmd/issues/262](https://github.com/microsoft/go-sqlcmd/issues/262)